spec/scenic/adapters/postgres_spec.rb

```diff
@@ -26,6 +26,14 @@ module Adapters
           expect(view.name).to eq("greetings")
           expect(view.materialized).to eq true
         end
+
+        it "raises an exception if the version of PostgreSQL is too old" do
+          adapter = Postgres.new
+          allow(adapter).to receive(:version_number) { 920 }
```

Comment:
You're stubbing the system under test here, which weakens the trust we can have in the test. Can we stub the `ActiveRecord::Base.connection` collaborator instead?

https://github.com/thoughtbot/scenic/pull/111/files#diff-5c0a9160f7877473c255109f580ef941R175